fin-advisor-crm/templates/partials/sidebar.html

```diff
@@ -37,6 +37,23 @@
                     Log Out
                 </a>
             </li>
+            <li class="sidebar-item">
+                <a href="#" class="sidebar-link collapsed" data-bs-target="#pages" data-bs-toggle="collapse"
+                    aria-expanded="false"><i class="fa-solid fa-people-roof pe-2"></i>
+                    Financial Markets
+                </a>
+                <ul id="pages" class="sidebar-dropdown list-unstyled collapse" data-bs-parent="#sidebar">
+                    <li class="sidebar-item">
+                        <a href="#" class="sidebar-link">Equities</a>
+                    </li>
+                    <li class="sidebar-item">
+                        <a href="#" class="sidebar-link">Fixed Income</a>
+                    </li>
+                    <li class="sidebar-item">
+                        <a href="#" class="sidebar-link">Currencies</a>
+                    </li>
```

Comment:
This should be before "Log Out"

Reply:
I have corrected the positioning of logout button